scripts/jenkins/qat/test.make

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for QAT-DPDK jenkins.
+
+qat: qat-checks qat-pull qat-cluster qat-plugin qat-tests
+
+qat-checks:
+	bash scripts/jenkins/qat/checks.sh
+
+qat-pull:
+	bash scripts/jenkins/qat/images-pull.sh
+
+qat-cluster:
+	bash scripts/jenkins/qat/k8s-install.sh
+
+qat-plugin:
+	bash scripts/jenkins/qat/plugin-deploy.sh
+
+qat-tests: qat-tc-crypto qat-tc-compress
+
+qat-tc-crypto:
+	TCNAME="crypto" TCNUM=1 bash ./scripts/jenkins/qat/tc-deploy.sh
+
+qat-tc-compress:
+	TCNAME="compress" TCNUM=1 bash ./scripts/jenkins/qat/tc-deploy.sh
```

Comment:
Nit: these targets are not marked as PHONY. Why not just have it as a separate Makefile and call the targets with `make -C scripts/jenkins/qat qat-tc-crypto`?

Reply:
@rojkov sure,

with last commit we can use `make -C scripts/jenkins/qat qat-tc-crypto` or `make qat-tc-crypto` directly in the dir as shown below:

```
dir(path: "$QAT_DIR") {
  sh 'make qat-tc-crypto'
}
```